medicinskij-academicheskij-jurnal.csl

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Медицинский академический журнал (Russian)</title>
+    <id>http://www.zotero.org/styles/medicinskij-academicheskij-jurnal</id>
+    <link href="http://www.zotero.org/styles/medicinskij-academicheskij-jurnal" rel="self"/>
+    <link href="http://protect.gost.ru/document.aspx?control=7&amp;id=173511" rel="documentation"/>
+    <author>
+      <name>Valery Mukhin</name>
+      <email>Valery.Mukhin@gmail.com</email>
+      <uri>http://sites.google.com/site/yurkin/</uri>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
+    <summary>This style is developed on the base of the &quot;gost-r-7-0-5-2008-numeric&quot  by Yurkin Maxim;, and the main difference is that in this style the delimiters are dots with dashs. And all authors are included in bibliograohies. !!! IMPORTANT !!! Citations are numeric, order of bibliography is in order of citations, default locale is 'en'.    РЎС‚РёР»СЊ РѕСЃРЅРѕРІР°РЅ РЅР° СЃС‚РёР»Рµ &quot;gost-r-7-0-5-2008-numeric&quot;. РџРѕСЏСЃРЅРµРЅРёСЏ Рє РёСЃС…РѕРґРЅРѕРјСѓ СЃС‚РёР»СЋ СЃР»РµРґСѓСЋС‰РёРµ: РћСЃРЅРѕРІРЅР°СЏ РїСЂРѕР±Р»РµРјР° СЃРµР№С‡Р°СЃ - СЌС‚Рѕ Р»РѕРєР°Р»РёР·Р°С†РёСЏ С‚РµСЂРјРёРЅРѕРІ (РЎ., Рў., РїРѕРґ СЂРµРґ.,Рё РґСЂ. ... РёР»Рё СЃРѕРѕС‚РІРµС‚СЃС‚РІРµРЅРЅРѕ P., Vol., ed., et al.). Р’ РїСЂРёРЅС†РёРїРµ СЌС‚Рѕ СЂР°Р±РѕС‚Р°РµС‚ Рё РєРѕРЅС‚СЂРѕР»Р»РёСЂСѓРµС‚СЃСЏ РїРѕР»РµРј  default-locale (РІС‹С€Рµ, РІРѕ РІС‚РѕСЂРѕР№ СЃС‚СЂРѕС‡РєРµ). Р”Р»СЏ СЂСѓСЃСЃРєРѕРіРѕ РЅР°РґРѕ Р·Р°РјРµРЅРёС‚СЊ РЅР° &quot;ru-RU&quot;. Р­С‚Рѕ РѕРїСЂРµРґРµР»СЏРµС‚ РѕРґРЅРё РЅР°СЃС‚СЂРѕР№РєРё РґР»СЏ РІСЃРµР№ Р±РёР±Р»РёРѕРіСЂР°С„РёРё. РћРґРЅР°РєРѕ, РёРґРµР°Р»СЊРЅРѕРµ СЂРµС€РµРЅРёРµ - СЌС‚Рѕ РІС‹Р±РёСЂР°С‚СЊ СЏР·С‹Рє Р»РѕРєР°Р»РёР·Р°С†РёРё С‚РµСЂРјРёРЅРѕРІ РІ Р·Р°РІРёСЃРёРјРѕСЃС‚Рё РѕС‚ СЏР·С‹РєР° Р·Р°РїРёСЃРё (РєР°Рє Рё РїСЂРµРґРїРёСЃС‹РІР°РµС‚ Р“РћРЎРў). Р­С‚Рѕ РЅРµРІРѕР·РјРѕР¶РЅРѕ СЃРґРµР»Р°С‚СЊ РІ СЂР°РјРєР°С… Zotero 4.0 Рё CSL 1.0, РЅРѕ СѓР¶Рµ СЃРµР№С‡Р°СЃ С‡С‚Рѕ-С‚Рѕ РјРѕР¶РЅРѕ СЃРґРµР»Р°С‚СЊ РІ СЂР°РјРєР°С… MultiLingual Zotero - СЃРј. РЅР°РїСЂРёРјРµСЂ http://forums.zotero.org/discussion/18482/multilingual-zotero-fields-and-csl-variables.    Р РїРѕСЃР»РµРґРЅРµРµ, РїР°С‚РµРЅС‚С‹ РїРѕ СѓРјРѕР»С‡Р°РЅРёСЋ РїСЂРµРґСЃС‚Р°РІР»РµРЅС‹ РєР°Рє USA, РїРѕСЃРєРѕР»СЊРєСѓ РІ CSL 1.0 РЅРµС‚ РїРµСЂРµРјРµРЅРЅРѕР№, СЃРѕРѕС‚РІРµС‚СЃС‚РІСѓСЋС‰РµР№ РїРѕР»СЋ country РІ Р±Р°Р·Рµ Zotero 4.0. РќР°РґРµСЋСЃСЊ, СЌС‚Рѕ Р±СѓРґРµС‚ РёСЃРїСЂР°РІР»РµРЅРѕ РІ Р±СѓРґСѓС‰РёС… РІРµСЂСЃРёСЏС….</summary>
```

Comment:
Dear Rintze,

Sorry for delay. I'm out of the office. I'll be back tomorrow, and I can
continue.
Yes this is in Russian. So was in the original style. This is important
information for Russian-speaking users on how to make Russian items were in
the beginning and then English. This is a requirement of Russian journals.

Valery Mukhin

2016-09-09 19:53 GMT+03:00 Rintze M. Zelle <notifications@github.com>:

> In medicinskij-academicheskij-jurnal.csl
> <https://github.com/citation-style-language/styles/pull/2214#discussion_r78212635>
> :
>
> > +<?xml version="1.0" encoding="utf-8"?>
> > +<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded">
> > +<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
> > +  <info>
> > +    <title>Медицинский академический журнал (Russian)</title>
> > +    <id>http://www.zotero.org/styles/medicinskij-academicheskij-jurnal</id>
> > +    <link href="http://www.zotero.org/styles/medicinskij-academicheskij-jurnal" rel="self"/>
> > +    <link href="http://protect.gost.ru/document.aspx?control=7&amp;id=173511" rel="documentation"/>
> > +    <author>
> > +      <name>Valery Mukhin</name>
> > +      <email>Valery.Mukhin@gmail.com</email>
> > +      <uri>http://sites.google.com/site/yurkin/</uri>
> > +    </author>
> > +    <category citation-format="numeric"/>
> > +    <category field="generic-base"/>
> > +    <summary>This style is developed on the base of the &quot;gost-r-7-0-5-2008-numeric&quot  by Yurkin Maxim;, and the main difference is that in this style the delimiters are dots with dashs. And all authors are included in bibliograohies. !!! IMPORTANT !!! Citations are numeric, order of bibliography is in order of citations, default locale is 'en'.    РЎС‚РёР»СЊ РѕСЃРЅРѕРІР°РЅ РЅР° СЃС‚РёР»Рµ &quot;gost-r-7-0-5-2008-numeric&quot;. РџРѕСЏСЃРЅРµРЅРёСЏ Рє РёСЃС…РѕРґРЅРѕРјСѓ СЃС‚РёР»СЋ СЃР»РµРґСѓСЋС‰РёРµ: РћСЃРЅРѕРІРЅР°СЏ РїСЂРѕР±Р»РµРјР° СЃРµР№С‡Р°СЃ - СЌС‚Рѕ Р»РѕРєР°Р»РёР·Р°С†РёСЏ С‚РµСЂРјРёРЅРѕРІ (РЎ., Рў., РїРѕРґ СЂРµРґ.,Рё РґСЂ. ... РёР»Рё СЃРѕРѕС‚РІРµС‚СЃС‚РІРµРЅРЅРѕ P., Vol., ed., et al.). Р’ РїСЂРёРЅС†РёРїРµ СЌС‚Рѕ СЂР°Р±РѕС‚Р°РµС‚ Рё РєРѕРЅС‚СЂРѕР»Р»РёСЂСѓРµС‚СЃСЏ РїРѕР»РµРј  default-locale (РІС‹С€Рµ, РІРѕ РІС‚РѕСЂРѕР№ СЃС‚СЂРѕС‡РєРµ). Р”Р»СЏ СЂСѓСЃСЃРєРѕРіРѕ РЅР°РґРѕ Р·Р°РјРµРЅРёС‚СЊ РЅР° &quot;ru-RU&quot;. Р­С‚Рѕ РѕРїСЂРµРґРµР»СЏРµС‚ РѕРґРЅРё РЅР°СЃС‚СЂРѕР№РєРё РґР»СЏ РІСЃРµР№ Р±РёР±Р»РёРѕРіСЂР°С„РёРё. РћРґРЅР°РєРѕ, РёРґРµР°Р»СЊРЅРѕРµ СЂРµС€РµРЅРёРµ - СЌС‚Рѕ РІС‹Р±РёСЂР°С‚СЊ СЏР·С‹Рє Р»РѕРєР°Р»РёР·Р°С†РёРё С‚РµСЂРјРёРЅРѕРІ РІ Р·Р°РІРёСЃРёРјРѕСЃС‚Рё РѕС‚ СЏР·С‹РєР° Р·Р°РїРёСЃРё (РєР°Рє Рё РїСЂРµРґРїРёСЃС‹РІР°РµС‚ Р“РћРЎРў). Р­С‚Рѕ РЅРµРІРѕР·РјРѕР¶РЅРѕ СЃРґРµР»Р°С‚СЊ РІ СЂР°РјРєР°С… Zotero 4.0 Рё CSL 1.0, РЅРѕ СѓР¶Рµ СЃРµР№С‡Р°СЃ С‡С‚Рѕ-С‚Рѕ РјРѕР¶РЅРѕ СЃРґРµР»Р°С‚СЊ РІ СЂР°РјРєР°С… MultiLingual Zotero - СЃРј. РЅР°РїСЂРёРјРµСЂ http://forums.zotero.org/discussion/18482/multilingual-zotero-fields-and-csl-variables.    Р˜ РїРѕСЃР»РµРґРЅРµРµ, РїР°С‚РµРЅС‚С‹ РїРѕ СѓРјРѕР»С‡Р°РЅРёСЋ РїСЂРµРґСЃС‚Р°РІР»РµРЅС‹ РєР°Рє USA, РїРѕСЃРєРѕР»СЊРєСѓ РІ CSL 1.0 РЅРµС‚ РїРµСЂРµРјРµРЅРЅРѕР№, СЃРѕРѕС‚РІРµС‚СЃС‚РІСѓСЋС‰РµР№ РїРѕР»СЋ country РІ Р±Р°Р·Рµ Zotero 4.0. РќР°РґРµСЋСЃСЊ, СЌС‚Рѕ Р±СѓРґРµС‚ РёСЃРїСЂР°РІР»РµРЅРѕ РІ Р±СѓРґСѓС‰РёС… РІРµСЂСЃРёСЏС….</summary>
>
> @ValeryMukhin <https://github.com/ValeryMukhin>, the summary shows a lot
> of text like "Р’ РїСЂРёРЅС†РёРїРµ СЌС‚Рѕ СЂР°Р±РѕС‚Р°РµС‚ Рё
> РєРѕРЅС‚СЂРѕР»Р»РёСЂСѓРµС‚СЃСЏ РїРѕР»РµРј". I assume this is supposed to be
> Russian (Cyrillic). Can you take a look?
>
> (if this is an encoding issue, changing the encoding in your source file
> to UTF8 should do the trick)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/citation-style-language/styles/pull/2214/files/9497f253fe55054bd58f6aa9f1180e41ee0035e2#r78212635>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AK6PJEHTwzlQYdjTqqIhOZ5ml78RnxgBks5qoY8ZgaJpZM4J49Um>
> .
>



-- 
Valery Mukhin, Ph.D.
Senior scientific researcher and scientific secretary,
Pavlov Dept. of Physiology
of State Institute for Experimental Medicine
Russian Academy of Medical Science.
Acad. Pavlov str., 12
St. Petersburg, 197376
Russia
Tel.: +7 921 362 0845;
E-mail: Valery.Mukhin@gmail.com